docs/variants/novacustom_ns5x_7x/releases.md

```diff
@@ -93,7 +93,7 @@ See how to verify signatures on [this video](https://asciinema.org/a/433461)
 
 ### Added
 
-- Support for NovaCustom NS5x
+- Support for NovaCustom NS5x/7x
```

Comment:
This is not the truth. Support for NS7x has been added in v1.1.0 of the Dasharo firmware

Reply:
Updated